nav.php

```diff
@@ -15,25 +15,25 @@
 	</a>
 	<div class="navbar-links">
 		<ul>
-			<li><a href="index.php">Home</a></li>
-			<li><a href>Playlists</a>
+			<li class="hover-underline-animation"><a href="index.php">Home</a></li>
```

Comment:
I added a hover technique that made an underline animation display when the user hovered over one of the nav links